README.md

```diff
@@ -25,7 +25,7 @@ Lighthouse is:
 - Security-focused. Fuzzing techniques have been continuously applied and several external security reviews have been performed.
 - Built in [Rust](https://www.rust-lang.org), a modern language providing unique safety guarantees and
 	excellent performance (comparable to C++).
-- Funded by various organisations, including Sigma Prime, the
+- Funded by various organizations, including Sigma Prime, the
```

Comment:
```suggestion
- Funded by various organisations, including Sigma Prime, the
```

Most of us are based in Australia, so we're committed to a mix of UK and US English. We hope our US audience understand anyway ;)